### PR TITLE
Fix for google.com

### PR DESCRIPTION
**Pull request by pnaf**
Fixes menu button on mobile view.

Before:
png

After:
png

### CODE REVIEW

**Comment by Gusted** [comment]
Hi!

Could you share a link whereby I can observe this behavior?

**Comment by pnaf** [comment]
Hi!

I can observe this behavior on https://www.google.com/ using a Firefox for Android user agent (e.g., `Mozilla/5.0 (Android 11; Mobile; rv:88.0) Gecko/88.0 Firefox/88.0`).

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -12772,6 +12772,7 @@ a[href*="about/products"][title]
 .JOmIqc
 .hLcKi
 #EcMbV
+#xRRFWe
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:12775]: I think a better fix is something along the lines:

```suggestion
#wrap a[href="/mobile/"]
```

So it doesn't rely on the obscure selector

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:12775]: Reference:

image

Comment (pnaf) [reply src/config/dynamic-theme-fixes.config]: Sounds good to me!

**Review by Gusted — APPROVED** [review]

**Pull request merged**